config/application.rb

```diff
@@ -18,7 +22,8 @@
 module PublishingAPI
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 5.1
+    config.load_defaults 6.0
+    config.autoloader = :classic
```

Comment:
It might be worth adding a `FIXME` comment here explaining why this was done, and how to fix it.